src/views/Home.vue

```diff
@@ -207,6 +208,20 @@ export default defineComponent({
     const canUndo = computed(() => store.getters[CAN_UNDO]);
     const canRedo = computed(() => store.getters[CAN_REDO]);
 
+    // add hotkeys
+    Mousetrap.bind(["ctrl+s"], () => {
```

Comment:
This process can be very time consuming, so it seems better to use export (ctrl+e) instead of save (ctrl+s).
```suggestion
    Mousetrap.bind(["ctrl+e"], () => {
```